### DISCUSSION TITLE
RAM only going to 1.5gb

### DISCUSSION THREAD

**Discussion by ghost** [plan-player-analytics/Plan#3814] [Q&A]
i'm running a docker minecraft server, and in the performance tab of the server's analytics, the ram is only going to 1.8 gb  While, in the docker container is going up to 4 gb pls help me

**Comment by AuroraLS3** [comment]
JVM restricts memory usage to 50% of OS MEMORY by default, you need to set JVM startup flag for the maximum allocation percentage

**Reply by ghost** [reply]
how do I do that

**Reply by AuroraLS3** [reply]
`java -XX:MaxRAMPercentage=60.0 -jar <jar>` (Replace 60.0 with a percentage you want to use)

**Reply by AuroraLS3** [reply]
If you can't affect java startup command directly you can use `JAVA_TOOL_OPTIONS="-XX:MaxRAMPercentage=60.0"` environment variable

**Reply by ghost** [reply]
Thank you :P
